packages/internal/src/storage-test-utils.js

```diff
@@ -184,14 +186,17 @@ export const makeFakeStorageKit = (rootPath, rootOptions) => {
               } catch (_err) {
                 streamCell = undefined;
               }
+              // StreamCells reset at block boundaries.
+              if (Number(streamCell?.blockHeight) !== currentBlockHeight) {
```

Comment:
```suggestion
              if (streamCell && Number(streamCell.blockHeight) !== currentBlockHeight) {
```

Reply:
Oops, good thing the test I had updated caught that :p